demo/index.html

```diff
@@ -204,6 +204,16 @@
         </div>
       </div>
 
+      <div class="hx-section exampleSection">
+        <div class="example">
+          <h3>Autocomplete Picker</h3>
+          <div id="autocomplete-picker"></div>
+          <script>
+            new hx.AutocompletePicker('#autocomplete-picker', ['Dave', 'Jane', 'Bob', 'Dan', 'Steve', 'Alejandro'])
+          </script>
+        </div>
+      </div>
```

Comment:
I thought this was missing on the demo page so I added it. I guess we should do that for other things that are missing as well at some point?